crates/driver/src/infra/api/routes/solve/dto/auction.rs

```diff
@@ -287,3 +301,9 @@ enum Class {
     Limit,
     Liquidity,
 }
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+enum FeePolicy {
+    QuoteDeviation { factor: f64, volume_cap_factor: f64 },
+}
```

Comment:
Shouldn't the dto also have documentation (so that it can be generated for the swagger docs)?